flash_examples/predict/semantic_segmentation.py

```diff
@@ -24,7 +24,9 @@
 )
 
 # 2. Load the model from a checkpoint
-model = SemanticSegmentation.load_from_checkpoint("semantic_segmentation_model.pt")
+model = SemanticSegmentation.load_from_checkpoint(
```

Comment:
Not sure what this has to do with the PR?

Reply:
It has a tiny bug, nothing major